samples/client/petstore/python-experimental/tests/test_pet_api.py

```diff
@@ -71,9 +71,10 @@ def request(self, *args, **kwargs):
 class PetApiTests(unittest.TestCase):
 
     def setUp(self):
-        config = Configuration()
-        config.host = HOST
-        self.api_client = petstore_api.ApiClient(config)
+        self.config = Configuration()
```

Comment:
Optionally pass in the host an access_token inputs into the Configuration
instance rather than setting them with .host .access_token